domains/authorization/Authorization.API/API/Models/Organization.cs

```diff
@@ -4,47 +4,36 @@
 
 namespace API.Models;
 
-public class Organization
+public class Organization : IEntity<Guid>
 {
     private Organization(
         Guid id,
-        IdpId idpId,
-        IdpOrganizationId idpOrganizationId,
         Tin tin,
         OrganizationName organizationName
     )
     {
         Id = id;
-        IdpId = idpId;
-        IdpOrganizationId = idpOrganizationId;
         Tin = tin;
-        OrganizationName = organizationName;
+        Name = organizationName;
     }
 
     private Organization()
     {
     }
 
     public Guid Id { get; private set; }
-    public IdpId IdpId { get; private set; } = null!;
-    public IdpOrganizationId IdpOrganizationId { get; private set; } = null!;
     public Tin Tin { get; private set; } = null!;
-    public OrganizationName OrganizationName { get; private set; } = null!;
-
+    public OrganizationName Name { get; private set; } = null!;
```

Comment:
We are using a Name Value Object now. Will this name object be used only for organizations, or for other entities? If it will be used for other entities, should we also use it for Users?

Reply:
You want one value object called Name? And be used for organisation, client and user?